readme.md

```diff
@@ -0,0 +1,11 @@
+# Filmweb Scraper
+### Setting Up
+#### Python version 3.8
+#### Install requirements
+`pip install -r requirements.txt`
+### Description
+##### This application scrapes specified data from the given number of pages from <www.filmweb.pl>.
+##### Function inside this program returns list of dictionaries with title, image, rate, genre and director if it exist.
+##### Otherwise, it will return an empty element.
+##### For PEP-8 styling checks run 
+`flake8 .`
```

Comment:
Hmm....
I think we don't need ##### for normal text. # Are for headers.
Here we probably want several lines/paragraphs
So...
## Description
This application scrapes specified data from the given number of pages from <www.filmweb.pl>.

Function inside this program returns list of dictionaries with title, image, rate, genre and director if it exist. Otherwise, it will return an empty element.
### Checks/Tools
For PEP-8 styling checks run 
```
flake8 .
```